packages/cloud_firestore/example/lib/main.dart

```diff
@@ -36,6 +36,8 @@ class MessageList extends StatelessWidget {
       stream: firestore.collection('messages').snapshots(),
       builder: (BuildContext context, AsyncSnapshot<QuerySnapshot> snapshot) {
         if (!snapshot.hasData) return const Text('Loading...');
+        print(snapshot.data.metadata.isFromCache);
```

Comment:
I would suggest removing these.

Reply:
Oops that was not supposed to make it into the PR, thanks!

Reply:
Done